coralproject/talk
# correct copy for flatten replies in configure

## What does this PR do?
Updates text in Configure regarding Flatten Replies to reflect new functionality

## What changes to the GraphQL/Database Schema does this PR introduce?
None.

## Does this PR introduce any new environment variables or feature flags? 
No.

## If any indexes were added, were they added to `INDEXES.md`?
n/a

## How do I test this PR?
Navigate to admin/configure/general, and note that in the setting for Flatten Replies, references to "four" have been replaced by "eight".
 
## How do we deploy this PR?
